plugins/profiles.koplugin/main.lua

```diff
@@ -69,7 +76,12 @@ function Profiles:getSubMenuItems()
                             text = _("Save"),
                             callback = function()
                                 local name = name_input:getInputText()
-                                if not self:newProfile(name) then
+                                if name == autostart_profile then
+                                    UIManager:show(InfoMessage:new{
+                                        text =  T(_("Reserved name can not be used: %1"), autostart_profile),
```

Comment:
(not sure about the phrase "cannot be used" but in any case ;-) )
```suggestion
                                        text =  T(_("Reserved name cannot be used: %1"), autostart_profile),
```